data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/NetworkService.kt

```diff
@@ -0,0 +1,57 @@
+package io.github.droidkaigi.feeder.data
+
+import io.github.droidkaigi.feeder.data.response.InstantSerializer
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.HttpClientEngineConfig
+import io.ktor.client.engine.HttpClientEngineFactory
+import io.ktor.client.features.defaultRequest
+import io.ktor.client.features.json.JsonFeature
+import io.ktor.client.features.json.serializer.KotlinxSerializer
+import io.ktor.client.features.logging.LogLevel
+import io.ktor.client.features.logging.Logger
+import io.ktor.client.features.logging.Logging
+import io.ktor.client.request.headers
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.modules.SerializersModule
+import kotlinx.serialization.modules.contextual
+
+class NetworkService private constructor(val httpClient: HttpClient) {
```

Comment:
Thank you for realizing the commonality!
I thought that this class would access the property of httpClient every time, and it seems that it is not good for encapsulation, so I thought that it is better to distribute HttpClient at present. 💭 
If you plan to make HttpClient a private property and encapsulate it in NetworkService, this is fine, but it seems that we will need to create comments and issues. 🙇

Reply:
However, it can be fixed immediately, and iOS development is prioritized over encapsulation issues, so we will merge this PR. 👍

Reply:
@takahirom  I thought it was a problem too.
I would like to improve it at another time.

Thank you for always reviewing!